pyorbital/tlefile.py

```diff
@@ -74,11 +74,14 @@ def read_platform_numbers(in_upper=False, num_as_int=False):
             parts = row.split()
             if len(parts) < 2:
                 continue
+            # The satellite name might have whitespace
+            platform = ' '.join(parts[:-1])
+            num = parts[-1]
```

Comment:
This doesn't work I think, examples from http://celestrak.com/NORAD/elements/weather.txt :
DMSP 5D-3 F16 (USA 172) 
EWS-G1 (GOES 13)
I'm not saying it was working better before though...

Reply:
It will. The `parts[-1]` is the platform number defined in `platforms.txt`.